<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QSpinBox { border: 3px inset grey; } 
QSpinBox::up-button { subcontrol-position: left; width: 32px; height: 35px;}
QSpinBox::down-button { subcontrol-position: right; width: 30px; height: 35px;}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(197, 197, 197);</string>
   </property>
   <widget class="QWidget" name="home_widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>581</width>
      <height>571</height>
     </rect>
    </property>
    <widget class="QPushButton" name="volume">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>267</width>
       <height>267</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#volume {
background-color: transparent;
border-image: url(:/Buttons/VolumeButtonBlack.png);
background: none;
border: none;
background-repeat: none;
}
#volume:pressed
{
border-image: url(:/Buttons/VolumeButtonWhite.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="channel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>267</width>
       <height>267</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#channel {
background-color: transparent;
border-image: url(:/Buttons/ChannelButtonBlack.png);
background: none;
border: none;
background-repeat: none;
}
#channel:pressed
{
border-image: url(:/Buttons/ChannelButtonWhite.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="power">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>267</width>
       <height>267</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#power {
background-color: transparent;
border-image: url(:/Buttons/PowerButtonBlack.png);
background: none;
border: none;
background-repeat: none;
}
#power:pressed
{
border-image: url(:/Buttons/PowerButtonWhite.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="volume_widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>581</width>
      <height>571</height>
     </rect>
    </property>
    <widget class="QPushButton" name="plus_volume">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>267</width>
       <height>267</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#plus_volume {
background-color: transparent;
border-image: url(:/Buttons/PlusButtonBlack.png);
background: none;
border: none;
background-repeat: none;
}
#plus_volume:pressed
{
border-image: url(:/Buttons/PlusButtonWhite.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="minus_volume">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>267</width>
       <height>267</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#minus_volume {
background-color: transparent;
border-image: url(:/Buttons/MinusButtonBlack.png);
background: none;
border: none;
background-repeat: none;
}
#minus_volume:pressed
{
border-image: url(:/Buttons/MinusButtonWhite.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="mute_volume">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>267</width>
       <height>267</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#mute_volume {
background-color: transparent;
border-image: url(:/Buttons/MuteButtonBlack.png);
background: none;
border: none;
background-repeat: none;
}
#mute_volume:pressed
{
border-image: url(:/Buttons/MuteButtonWhite.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>plus_volume</zorder>
    <zorder>minus_volume</zorder>
    <zorder>mute_volume</zorder>
    <zorder>label_2</zorder>
   </widget>
   <widget class="QPushButton" name="home">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>290</y>
      <width>267</width>
      <height>267</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#home {
background-color: transparent;
border-image: url(:/Buttons/HomeButtonBlack.png);
background: none;
border: none;
background-repeat: none;
}
#home:pressed
{
border-image: url(:/Buttons/HomeButtonWhite.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="channel_widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>580</width>
      <height>571</height>
     </rect>
    </property>
    <widget class="QPushButton" name="minus_channel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>267</width>
       <height>267</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#minus_channel {
background-color: transparent;
border-image: url(:/Buttons/MinusButtonBlack.png);
background: none;
border: none;
background-repeat: none;
}
#minus_channel:pressed
{
border-image: url(:/Buttons/MinusButtonWhite.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="plus_channel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>267</width>
       <height>267</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#plus_channel {
background-color: transparent;
border-image: url(:/Buttons/PlusButtonBlack.png);
background: none;
border: none;
background-repeat: none;
}
#plus_channel:pressed
{
border-image: url(:/Buttons/PlusButtonWhite.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="favorite_channel">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>267</width>
       <height>267</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#favorite_channel {
background-color: transparent;
border-image: url(:/Buttons/FavoriteButtonBlack.png);
background: none;
border: none;
background-repeat: none;
}
#favorite_channel:pressed
{
border-image: url(:/Buttons/FavoriteButtonWhite.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>16</width>
      <height>551</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>30</y>
      <width>291</width>
      <height>521</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(148, 148, 148);</string>
    </property>
    <property name="title">
     <string>Control</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>360</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>360</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>stop</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="freemode_radioButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Free</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="acquisition_radioButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Acquisition</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="p300remote_radioButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>P300 Remote</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
    <widget class="QSpinBox" name="frequency_Bspin">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>270</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(148, 148, 148);</string>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="showGroupSeparator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="frequency_Aspin">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>320</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>30</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Mode :</string>
     </property>
    </widget>
    <widget class="QLabel" name="frequency_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Frequency :</string>
     </property>
    </widget>
    <widget class="QLabel" name="frequency_Blabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>260</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Before Time :</string>
     </property>
    </widget>
    <widget class="QLabel" name="frequency_Alabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>309</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>After Time :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>400</y>
       <width>121</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="ressources.qrc">:/Buttons/polycortex-logo-1--1- (1).png</pixmap>
     </property>
    </widget>
    <zorder>pushButton</zorder>
    <zorder>pushButton_2</zorder>
    <zorder>freemode_radioButton</zorder>
    <zorder>acquisition_radioButton</zorder>
    <zorder>p300remote_radioButton</zorder>
    <zorder>label</zorder>
    <zorder>frequency_label</zorder>
    <zorder>frequency_Blabel</zorder>
    <zorder>frequency_Alabel</zorder>
    <zorder>frequency_Bspin</zorder>
    <zorder>frequency_Aspin</zorder>
    <zorder>label_2</zorder>
   </widget>
   <zorder>channel_widget</zorder>
   <zorder>home_widget</zorder>
   <zorder>volume_widget</zorder>
   <zorder>home</zorder>
   <zorder>line</zorder>
   <zorder>groupBox</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
